data/Debian-family.yaml

```diff
@@ -0,0 +1,3 @@
+---
+mongodb::mongos::service_user: 'mongodb'
+mongodb::mongos::service_group: 'mongodb'
```

Comment:
Please add the missing newline at the end.

Reply:
Removed the unused init.d template as well